ReactiveCocoa/Swift/Scheduler.swift

```diff
@@ -451,4 +451,20 @@ public final class TestScheduler: DateSchedulerType {
 	public func run() {
 		advanceToDate(NSDate.distantFuture())
 	}
+	
+	/// Retreats the virtualized clock by the given interval.
+	/// This simulates that user changes device date.
+	///
+	/// - parameters:
+	///   - interval: Interval by which the current date will be retreated.
+	public func retreatCurrentDateByInterval(interval: NSTimeInterval) {
```

Comment:
I think the naming of this method is a bit off. It doesn't match `advanceByInterval` because it includes `CurrentDate`, but I also don't love the verb _retreat_ in this context.

What would you think of renaming this `rewindByInterval`?